docs/modules/ROOT/pages/auth/setup.adoc

```diff
@@ -3,44 +3,50 @@
 
 == Configuration
 
-If you want the Neo4j GraphQL Library to perform JWT decoding and verification for you, you must pass the configuration option `jwt` into the `Neo4jGraphQL` or `OGM` constructor, which has EITHER of the following arguments:
-
-- `jwksEndpoint` (recommended) - The OpenID Public Key Set Endpoint that verifies the JWT. The https://auth0.com/docs/security/tokens/json-web-tokens/json-web-key-sets[JSON Web Key Sets] used under the OpenID Configuration are typically stored at an endpoint similar to \https://YOUR_DOMAIN/.well-known/jwks.json
-- `secret` - The secret to be used to decode and verify JWTs. Either the secret or the PEM encoded public key used to verify the JWT.
-
-Optional arguments:
-
-- `noVerify` (optional) - Disable verification of JWTs, defaults to _false_
-- `rolesPath` (optional) - A string key to specify where to find roles in the JWT, defaults to "roles"
-
-The recommended construction of a `Neo4jGraphQL` instance would be:
+To get started with auth you must first create your own auth plugin. For most use cases you will only need to use our provided plugins at `@neo4j/graphql-plugin-auth`. Below is a basic example using the `Neo4jGraphQLAuthJWTPlugin` class: 
```

Comment:
The first sentence here makes it sound like users have to write their own plugin, can we change the wording a bit?

```suggestion
To get started with auth you need an instance of an auth plugin for the Neo4j GraphQL Library. For most use cases you will only need to use our provided plugins at `@neo4j/graphql-plugin-auth`. Below is a basic example using the `Neo4jGraphQLAuthJWTPlugin` class: 
```